libs/core/execution/CMakeLists.txt

```diff
@@ -87,6 +88,7 @@ set(execution_compat_headers
     hpx/parallel/executors/execution_parameters.hpp => hpx/execution.hpp
     hpx/parallel/executors/fused_bulk_execute.hpp => hpx/execution.hpp
     hpx/parallel/executors/guided_chunk_size.hpp => hpx/execution.hpp
+    hpx/parallel/executors/num_cores.hpp => hpx/execution.hpp
```

Comment:
This is not needed. We have this section for backwards compatibility only.

Reply:
> 

thanks, removed line91